schema/trivia.schema.json

```
@@ -0,0 +1,52 @@
+{
+  "$id": "https://raw.githubusercontent.com/Cog-Creators/Red-DiscordBot/V3/develop/schema/trivia.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Red-DiscordBot Trivia List file",
+  "type": "object",
+  "properties": {
+    "AUTHOR": {
+      "type": "string",
+      "description": "Author of the Trivia quiz."
+    },
+    "CONFIG": {
+      "type": "object",
+      "description": "The configuration for a trivia session.",
+      "properties": {
+        "bot_plays": {
+          "type": "boolean",
+          "description": "Whether or not the bot gains points during the session. Defaults to true."
+        },
+        "delay": {
+          "type": ["integer", "number"],
+          "description": "The maximum number of seconds permitted to answer a question, must be a positive number greater than or equal to 4.0. Defaults to 15.0.",
+          "minimum": 4.0
+        },
+        "max_score": {
+          "type": "integer",
+          "description": "Number of points required in order to win the trivia, must be a positive integer. Defaults to 10.",
+          "exclusiveMinimum": 0
+        },
+        "payout_multiplier": {
+          "type": ["integer", "number"],
```

`number` already supports integers:
```suggestion
          "type": "number",
```